adafruit_sgp30.py

```diff
@@ -168,6 +169,19 @@ def set_iaq_humidity(self, gramsPM3):  # pylint: disable=invalid-name
             buffer += arr
         self._run_profile(["iaq_set_humidity", [0x20, 0x61] + buffer, 0, 0.01])
 
+    def set_iaq_relative_humidity(self, celcius, relative_humidity):
+        """
+        Set the humidity in g/m3 for eCo2 and TVOC compensation algorithm. The absolute humidity is calculated from the temperature and relative humidity (as a percentage).
```

Comment:
Looks like this docstring should be broken up into multiple lines:

```suggestion
        Set the humidity in g/m3 for eCo2 and TVOC compensation algorithm.
        The absolute humidity is calculated from the temperature and relative humidity
        (as a percentage).
```

Reply:
This surprised me at first, so I checked and Sphinx only makes a new line / paragraph after a double new line, so it's safe to wrap this manually - amended.